src/lxc/tools/lxc_start.c

```diff
@@ -231,6 +231,14 @@ int main(int argc, char *argv[])
 		exit(err);
 	lxc_log_options_no_override();
 
+	if (geteuid()) {
```

Comment:
Maybe, but since the goal is better error reporting, is there any point in hiding this check behind uid != 0?  Is there any case where the access check fails when getuid returns 0, where we would *not* want to give a clear error message and exit now?